Bottom: 701bfad47cfeefb22f51e3977fe3629945359257
Top:    dd3e337c812c828bd16fb07d95a805d5eaaccc91
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-07-23 16:28:11 -0700

Refresh of fork-xfer

---

diff --git a/src/preload.c b/src/preload.c
index 82a4125..79340c6 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -536,8 +536,8 @@ static void fork_passive(int socket)
 		goto lclose;
 
 	msg = 0;
-	ret = real.write(sfd, &msg, sizeof msg);
-	if (ret != sizeof msg)
+	len = real.write(sfd, &msg, sizeof msg);
+	if (len != sizeof msg)
 		goto lclose;
 
 	dfd = raccept(lfd, NULL, NULL);
